libs/application/templates/vmst/confirm-job-search/README.md

```
@@ -0,0 +1,42 @@
+**Code owner:** Origo
+
+## Gervimenn
+
+Thelma Hrönn ÞÍ Sveinsdóttir - 010455-1469
+Note: Gervimenn must have active unemployment benefits registered in VMST. Applications can only be submitted between the 20th and 25th of each month, with a grace period extending to the 4th of the following month.
+
+-
+
+## State flow
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)